forge-game/src/main/java/forge/game/ability/AbilityFactory.java

```diff
@@ -63,7 +63,8 @@ public final class AbilityFactory {
             "ChooseSubAbility", // Can choose a player via ChoosePlayer
             "CantChooseSubAbility", // Can't choose a player via ChoosePlayer
             "AnimateSubAbility", // For ChangeZone Effects to Animate before ETB
-            "ReturnAbility" // for Delayed Trigger on Magpie
+            "ReturnAbility", // for Delayed Trigger on Magpie
+            "W", "U", "B", "R", "G" // for Specialize
```

Comment:
this looks like boilerplate, I wonder if it should just be hardcoded?

It might also be better if this could be done more like other double faced cards so you can write them each cleaner and also have the unique Oracles?

Reply:
I'd much rather hardcode it, just need to figure out how. Executing a certain SVar from inside the Specialize effect is new to me and this was the only way I could get it to work. I tried a bunch of other things, too. Is there a good example? I feel like the answer might be related to GenericChoice or Charm, but with a specified list. I'll look into it more, if you have any quick pointers please share 😁 

I definitely like the idea of doing it more like dfcs, but I can't get my head wrapped around
 - a. how to start supporting multiple back sides (strange phrase out of context 😆 )
 - b. how to make it perpetual – right now all perpetual stuff is forced with the effects